aten/src/ATen/core/Type.h

```diff
@@ -140,422 +140,6 @@ struct CAFFE2_API Type {
     }
   }
 
-  // example
```

Comment:
What's blocking removing `Type.h` entirely?

Reply:
A few functions like unsafeTensorFromTH are still on Type and haven't found a new home yet.